core/dungeon-masters-guide/items/items-weapons.xml

```diff
@@ -538,6 +538,212 @@
+			<p class="indent">The weapon is bonded to the family line it is meant to serve. Once it has bonded with an owner who shares its ideals, its loyalty is absolute.</p>
+			<p class="indent">If a <em>moonblade</em> has a flaw, it is overconfidence. Once it has decided on an owner, it believes that only that person should wield it, even if the owner falls short of elven ideals.</p>
+		</description>
+		<setters>
+			<set name="keywords">longsword</set>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="longsword">Weapon</set>
+			<set name="attunement" addition="by an elf or half-elf of neutral good alignment">true</set>
+			<set name="rarity">Legendary</set>
+			<set name="enhancement">1</set>
+			<set name="weapon">Longsword</set>
+		</setters>
+	</element>
+	<element name="Blackrazor" type="Magic Item" source="Dungeon Master’s Guide" id="ID_WOTC_DMG_MAGIC_ITEM_BLACKRAZOR">
+		<description>
+			<p>Hidden in the dungeon of White Plume Mountain, <em>Blackrazor</em> shines like a piece of night sky filled with stars. Its black scabbard is decorated with pieces of cut obsidian.</p>
+			<p class="indent">You gain a +3 bonus to attack and damage rolls made with this magic weapon. It has the following additional properties.</p>
+			<p class="indent"><i><b>Devour Soul.</b></i> Whenever you use it to reduce a creature to 0 hit points, the sword slays the creature devours its soul, unless it is a construct or an undead. A creature whose soul has been devoured by <em>Blackrazor</em> can be restored to life only by a <i>wish</i> spell.</p> 
+			<p class="indent">When it devours a soul, <em>Blackrazor</em> grants you temporary hit points equal to the slain creature's hit point maximum. These hit points fade after 24 hours. As long as these temporary hit points last and you keep <em>Blackrazor</em> in hand, you have advantage on attack rolls, saving throws, and ability checks.</p> 
+			<p class="indent">If you hit an undead with this weapon, you take 1d10 necrotic damage and the target regains 1d10 hit points. If this necrotic damage reduces you to 0 hit points, <em>Blackrazor</em> devours your soul.</p>
+			<p class="indent"><i><b>Soul Hunter.</b></i> While you hold the weapon, you are aware of the presence of Tiny or larger creatures within 60 feet of you that aren't constructs or undead. You also can't be charmed or frightened.</p>
+			<p class="indent"><em>Blackrazor</em> can cast the <i>haste</i> spell on you once per day. It decides when to cast the spell and maintains concentration on it so that you don't have to.</p>
+			<p class="indent"><i><b>Sentience.</b></i> <em>Blackrazor</em> is a sentient chaotic neutral weapon with an Intelligence of 17, a Wisdom of 10, and a Charisma of 19. It has hearing and darkvision out to a range of 120 feet.</p> 
```

Comment:
for some reason doing it with `<em>` tags causes app to ignore space between the tags.
```suggestion
			<p class="indent"><i><b>Sentience.</b> Blackrazor</i> is a sentient chaotic neutral weapon with an Intelligence of 17, a Wisdom of 10, and a Charisma of 19. It has hearing and darkvision out to a range of 120 feet.</p> 
```